velox/exec/MergeJoin.cpp

```diff
@@ -321,6 +328,13 @@ void MergeJoin::addOutputRow(
     }
   }
 
+  // Anti join needs to track the left side rows that have no match on the
+  // right.
+  if (isAntiJoin(joinType_) && !filter_ && leftJoinTracker_) {
```

Comment:
should we here just assert that `leftJoinTracker_` exists instead?
```
if (isAntiJoin(joinType_)) {
  VELOX_CHECK(leftJoinTracker_);
```

Reply:
@pedroerp Updated.